Acrolinx.Sidebar/AcrolinxSidebar.cs

```diff
@@ -124,20 +127,16 @@ private void SetDefaultClientLocaleIfNotSet()
             }
         }
 
-        private void SetDefaultForSidebarLocationAndShowServerSelectorIfLocationNotSet(string serverAddress)
+        private string GetStartPageURL()
         {
-            if (string.IsNullOrWhiteSpace(this.SidebarSourceLocation))
+            var assemblyLocation = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + @"\Acrolinx.Startpage.dll";
```

Comment:
will it be there in every case?

Reply:
Yes it has to be there. And Sidebar project or other plugin project should copy it to execution directory.

Reply:
What if that's not possible? Any way of workaround? Or do we just change the SDK when it happens?

Reply:
Creating case for this.